addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -485,6 +485,7 @@ body.editor_enable.editor_has_snippets {
 
         we-button, we-toggler {
             @extend %we-generic-button;
+            border:0;
```

Comment:
This will remove borders from any button/dropdown  :/ 
![image](https://user-images.githubusercontent.com/8558351/135295463-2c28ff8e-b3a7-451a-84a8-99432cae8d52.png)

Reply:
My bad @stefanorigano , copy-paste mistake, should had been `border-radius` instead of `border`.
Pushed again. :)

Reply:
I see why you did that, but now <select> fields have no border-radius ^^
![image](https://user-images.githubusercontent.com/8558351/135437436-2fa76cf0-c056-4c61-8016-44b6e8fefe7e.png)